server/controllers/entities/contributions.js

```diff
@@ -6,7 +6,7 @@ const responses_ = __.require('lib', 'responses')
 const patches_ = require('./lib/patches')
 
 const sanitization = {
-  user: {},
+  user: { optional: true },
   limit: { default: 100, max: 10000 },
```

Comment:
maybe 10 000 is too much. It made sense for a single entity, but for all of them, that's only a fraction of the exiting changes and, at the same time, this page could quickly become quite intense for the server to load all those changes.
bonus point : define recent in the branch name `recent-changes` :)

Reply:
@jum-s how much then? 1000?

Reply:
there is a reason why i did not suggest any figure. I guess 1000 is fine

Reply:
done in 52b7368e